.github/workflows/set-milestone-on-referenced-issue.yml

```diff
@@ -0,0 +1,33 @@
+#############################################################################
+# GitHub action to set latest milestone on issue of merged PR.
+#
+#############################################################################
+name: 'Set milestone on issue of merged PR'
+
+on:
+  pull_request:
+    types:
+      - closed
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.event.pull_request.number || github.ref }}
+  cancel-in-progress: true
+
+permissions:
+  issues: write
+  pull-requests: write
+
+jobs:
+  set-milestone:
+    name: 'Set milestone on issue of merged PR #${{ github.event.pull_request.number }}'
```

Comment:
unrelated:
I like a fact that action execution prints input in its title https://github.com/stoyanK7/checkstyle/actions/workflows/set-milestone-on-referenced-issue.yml
how you did this and how can we make it for our release executions ?

All are same: https://github.com/checkstyle/checkstyle/actions/workflows/bump-version-and-update-milestone.yml

Reply:
We already do the same with release prepare and perform:
https://github.com/checkstyle/checkstyle/actions/runs/4272544721/jobs/7437786435
`Release perform 10.8.0`  
https://github.com/checkstyle/checkstyle/blob/ca650680735f305651a4fe3663f854123b89a3b5/.github/workflows/maven-release-perform.yml#L24

Reply:
> how you did this and how can we make it for our release executions ?

Which release executions specifically? All of them?

Reply:
Action does not print any input in title:
![Screenshot from 2023-03-01 17-04-29](https://user-images.githubusercontent.com/23459549/222195100-cab9b957-69b9-4b23-9681-2763ad56f08a.png)
`#9` is the ninth job that I ran
`Pull request #52` is automatically put by github.

However, individual jobs do:
![Screenshot from 2023-03-01 17-04-52](https://user-images.githubusercontent.com/23459549/222195256-750f5afb-f895-4a42-b72e-84c614cf0b75.png)
`Set milestone on issue of merged PR #52`

Reply:
@romani 

However, there is this: https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#run-name
> The name for workflow runs generated from the workflow. GitHub displays the workflow run name in the list of workflow runs on your repository's "Actions" tab.

I think we can use it to accomplish what you want.

Reply:
Please create issue on this to do separately

Reply:
Done.